app/views/users/invitations/new.html.erb

```diff
@@ -2,19 +2,27 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= link_to "Back to list", team_members_path, class: "govuk-back-link" %>
-    <h2 class="govuk-heading-l">
-      Invite a team member
-    </h2>
-
-    <div class="govuk-details">
-      <div class="govuk-details__text">
-        <p class="govuk-body">
-          Team members will receive an email invitation to join your organisation's GovWifi Admin dashboard.
-        </p>
+    <% if current_user.super_admin? %>
+      <%= link_to "Back to organisation", admin_organisation_path(params[:id]), class: "govuk-back-link" %>
+      <h2 class="govuk-heading-l">Invite a team member to <%= params[:name] %> </h2>
```

Comment:
can the target organisation be exposed as an object in the controller? This will allow for not needing to pass the org name around in the path